german/Korallen/Hoehle/Unterwasserstation/unterwasserstation.md

```diff
@@ -0,0 +1,4 @@
+Die Unterwasserstation scheint auf der Unterseite eine mit Luft gefüllte Auftauchstation für U-Boote zu haben.
+Da Sie immer noch nicht geatmet haben und der Weg aus der Höle zu weit ist, sind Sie gezwungen, in die Station zu tauchen.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.